include/libvfio-user.h

```diff
@@ -655,7 +655,9 @@ vfu_unmap_sg(vfu_ctx_t *vfu_ctx, const dma_sg_t *sg,
              struct iovec *iov, int cnt);
 
 /**
- * Read from the dma region exposed by the client.
+ * Read from the dma region exposed by the client. This can be used as an
+ * alternative to vfu_map_sg(), if the region is not directly mappable, or DMA
+ * callbacks have not been provided.
```

Comment:
Should we be more clear and call it as "DMA notification callbacks"

Reply:
done